src/styles/index.css

```
@@ -0,0 +1,6 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import './typography.css';
```

Biome (2.1.2)

[error] This `@import` is in the wrong position.

Any `@import` rules must precede all other valid at-rules and style rules in a stylesheet (ignoring `@charset` and `@layer`), or else the `@import` rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)